### DISCUSSION TITLE
SFC Loader on demand compile .vue file on server & reuse output

### DISCUSSION THREAD

**Discussion by LucaGabi** [FranckFreiburger/vue3-sfc-loader#199] [Q&A]
Hi, I'm not familiar with vue3-sfc-loader but I understood that it's able to load .vue files with no prior build step on client side. My goal is to be able to have plugins that can be loaded at runtime but only need to be compiled once on the server side or only if plugin files are changed.

Can I use vue3-sfc-loader to compile and output a .js file (on the server) that can be loaded at runtime (on-demand) in vue app and not need the vue3-sfc-loader on the client side ?

**Comment by FranckFreiburger** [comment]
Hi,
Unfortunately, it is not possible.
It is possible to compile and cache your plugins (.vue) server-side (see [server-side compilation & cache](https://github.com/FranckFreiburger/vue3-sfc-loader/discussions/178)), but `vue3-sfc-loader` will still be needed client-side.
